### PR TITLE
Update certificates used by JavaScript browser tests

### PR DESCRIPTION
**Pull request by pepone**

### CODE REVIEW

File: certs/client_key.pem
Comment (pepone) [inline certs/client_key.pem:1]: Removed, it was not used.

File: certs/server_key.pem
Comment (pepone) [inline certs/server_key.pem:1]: Updated the key too much the updated certificates, and remove the password. The makecerts.py script can be updated to deal with this later.

**Comment by externl** [comment]
Sorry, I'm not following. How are these two certs related to JS?

**Comment by pepone** [comment]
server_key is used by the HTTP server, run by the tests scripts

https://github.com/zeroc-ice/ice/blob/a2cbac19b054bd08d8c980d77272da295f9348ce/scripts/Util.py#L3027

`client_key.pem` was just unused and removed it.

**Review by externl — APPROVED** [review]

**Pull request merged**